processout-sdk/src/androidTest/java/com/processout/processout_sdk/UITestSuite.java

```diff
@@ -145,10 +155,11 @@ public void testSuccessful3DS2Payment() {
 
         final Activity withActivity = activityRule.getActivity();
         final ProcessOut p = new ProcessOut(withActivity, projectId);
-        Card c = new Card("4000000000003253", 10, 20, "737");
-        p.tokenize(c, null, new TokenCallback() {
+        Card paymentCard = new Card("4000000000003253", cardExpirationMonth, cardExpirationYear, "737");
+        p.tokenize(paymentCard, null, new TokenCallback() {
             @Override
             public void onError(Exception error) {
+                // todo: :adamszewe: [16/02/21] -
```

Comment:
@adamszewe - just checking if this was left here by accident.

Reply:
oh, yeah :sweat: , fixing